forge-gui/res/cardsfolder/upcoming/back_for_seconds.txt

```diff
@@ -0,0 +1,11 @@
+Name:Back for Seconds
+ManaCost:2 B
+Types:Sorcery
+K:Bargain
+A:SP$ Pump | Origin$ Graveyard | TgtZone$ Graveyard | RememberTargets$ True | TargetMin$ 0 | TargetMax$ 2 | TgtPrompt$ Choose up to two target creature card in your graveyard | ValidTgts$ Creature.YouOwn | SubAbility$ DBCheat | SpellDescription$ Return up to two target creature cards from your graveyard to your hand. If this spell was bargained, you may put one of those cards with mana value 4 or less onto the battlefield instead of putting it into your hand.
+SVar:DBCheat:DB$ ChangeZone | ChangeType$ Card.IsRemembered+cmcLE4 | OptionalPrompt$ Do you want to put one of those cards with mana value 4 or less onto the battlefield instead of putting it into your hand ? | Optional$ True | Origin$ Graveyard | Destination$ Battlefield | Hidden$ True | Condition$ Bargain | ChangeNum$ 1 | SubAbility$ DBChangeAll
+SVar:DBChangeAll:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered | Origin$ Graveyard | Destination$ Hand | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRememberbed$ True
```

Comment:
typo

Reply:
better rewrite with Defined

Reply:
the DBCheat probably need a Forget remembered?

or is that already handled by timestamp checks?

Reply:
Defined for ChangeZoneAll is for a defined player I think?

Reply:
just avoid Remembering

Reply:
Problem now is that you can only choose one card (even if both creatures have cmc le4)

Reply:
Try `Defined$ Targeted.cmcLE4`

Reply:
hmm still doesn't let me choose.

Reply:
+ `ChooseFromDefined$`

Reply:
thanks working now !